project_euler/problem_112/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
```suggestion
```

Reply:
This won't work just like I told you before

Reply:
Delete the file and create a new file without adding anything to it (not even the newline).
From the terminal: `touch __init__.py`

Reply:
I have tried this before on my previous PR's and it wouldn't work. Mr Clauss has acknowledged it as well. I think its best if I revert this change back.

Reply:
Please try it. I created around 62 empty `__init__.py` files and nothing failed and if possible start using `pre-commit`, it will fix all these small errors for you.